usecases/replica/replicator.go

```diff
@@ -83,7 +83,8 @@ func NewReplicator(className string,
 		client:      client,
 		resolver:    resolver,
 		log:         l,
-		Finder:      NewFinder(className, resolver, client, l),
+		Finder: NewFinder(className, resolver, client, l,
```

Comment:
**question:** is `defaultPullBackOffInitialInterval`, `defaultPullBackOffMaxElapsedTime` planned to be configurable via env vars ?

Reply:
Yes that was my thought exactly! Would do so in a follow up PR